tests/test_responses.py

```diff
@@ -905,7 +905,9 @@ def __init__(self) -> None:
             self.might_be_secret = "iunisud109283012"
 
     with pytest.raises(TypeError):
-        json(NotSerializable())
+        response = json(NotSerializable())
+        # NOTE: it's a trade-off to serialize at content level instead of serialize at json function call
```

Comment:
But, after re-writing i changed one of tests, because now serialization to json can be proceeded only when `response.content.body` is accesed